### PR TITLE
fix ccat

### PR DESCRIPTION
**Pull request by bernt-matthias**
- using test data from original source
- removed python wrapper
- nasty hack to test with chromInfo (same as somewhere in IUC)

### CODE REVIEW

**Comment by bgruening** [comment]
That is a lot of changes, should it move to IUC?

**Comment by bernt-matthias** [comment]
Not sure .. but I do not care much. Is the tool in use, e.g on usegalaxy?

**Comment by bgruening** [comment]
If you use it, this is enough to maintain it I would say.

**Comment by bernt-matthias** [comment]
Actually I never used it :) I just have seen that it failed CI .. or it just was it `.tt_skip` ..

**Comment by nsoranzo** [comment]
Can you fix the conflict please?

**Comment by nsoranzo** [comment]
I think the merge was wrong, you still need to remove ccat from `.tt_skip`

**Comment by bernt-matthias** [comment]
ready to go.

**Comment by bernt-matthias** [comment]
Anyone wants to approve/merge this?

File: tools/ccat/ccat_wrapper.xml [hunk 1]
```diff
@@ -1,32 +1,49 @@
-<tool id="peakcalling_ccat" name="CCAT" version="0.0.1">
+<tool id="peakcalling_ccat" name="CCAT" version="0.0.2">
```
[the remark this answers is not in the document]
Comment (bernt-matthias) [reply tools/ccat/ccat_wrapper.xml:1]: done

File: tools/ccat/ccat_wrapper.xml [hunk 2]
```diff
@@ -54,23 +71,24 @@
         <param name="bootstrap_pass" type="integer" label="Number of passes in the bootstrapping process" value="50"/>
       </when>
     </conditional>
+    <param name="in_test_mode" type="hidden" value="no" />
```
[the remark this answers is not in the document]
Comment (bernt-matthias) [reply tools/ccat/ccat_wrapper.xml:74]: It appears to me that the `chromInfo` variable may not be usable in tests https://github.com/galaxyproject/tools-devteam/blob/8163a1ac128c722246d12b66841b3df91e38594f/tools/ccat/ccat_wrapper.xml#L17

When executing the tool `chromInfo` is evaluated to `GALAXY_ROOT/tool-data/shared/ucsc/chrom/hg18.len` which does not exist (also hg18 is distinct from the dbkey of the input). 

All quite mysterious.. and I doubt that this works except for a few installations. 

Isn't there a data table with chromosome length infos?

File: tools/ccat/ccat_wrapper.xml [hunk 3]
```diff
@@ -79,78 +97,45 @@
...
-bootstrapPass	${options_type[ 'bootstrap_pass' ]}
-#end if</configfile>
-  </configfiles>
   <tests>
     <test>
-      <param name="input_tag_file" value="chipseq_enriched.bed.gz" ftype="bed" dbkey="hg18" />
-      <param name="input_control_file" value="chipseq_input.bed.gz" ftype="bed" dbkey="hg18" />
+      <param name="input_tag_file" value="ES_CTCF_chr19.bed" ftype="bed" dbkey="hg18" />
+      <param name="input_control_file" value="ES_GFP_chr19.bed" ftype="bed" dbkey="hg18" />
+      <param name="chromInfo" value="len.tsv" ftype="tabular" dbkey="hg18" />
```
[the remark this answers is not in the document]
Comment (bernt-matthias) [reply tools/ccat/ccat_wrapper.xml:104]: Yeahm this can be removed. Its not used in the test anyway. Also see my other comment.

**Review by bgruening — APPROVED** [review]

**Pull request merged**